core/src/OCP/accessibility.js

```diff
@@ -22,11 +22,23 @@
 
 import { loadState } from '@nextcloud/initial-state'
 
+/**
+ * Set the page heading
+ *
+ * @param {string} [heading] page title from the history api
+ */
```

Comment:
nit

```suggestion
 * @since 27.0.0
 */
```

although this should probably be dropped from the future backport PR